source/contracts/trading/ClaimProceeds.sol

```diff
@@ -51,11 +56,19 @@ contract ClaimProceeds is CashAutoConverter, ReentrancyGuard, IClaimProceeds {
             if (_reporterShare > 0) {
                 require(_denominationToken.transferFrom(_market, _market.getUniverse().getNextReportingWindow(), _reporterShare));
             }
+            
+            _numShares = _numShares.add(_numberOfShares);
+            _numPayoutTokens = _numPayoutTokens.add(_shareHolderShare);
         }
 
+        logProceedsClaimed(_market, msg.sender, _numShares, _numPayoutTokens);
```

Comment:
It feels like this should be called for each share token type that has non-zero value.  It also feels like the event should contain the address of the share being redeemed.  cc @tinybike @epheph

Reply:
Agreed, IMO `logProceedsClaimed` should be called once per iteration (of the loop above it).